core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -211,6 +213,22 @@ public void testFromXContent() throws IOException {
         }
     }
 
+    protected Tuple<MapperService, SB> mockMapperServiceAndSuggestionBuilder(
```

Comment:
I had another go at this (thought I posted the link already yesterday, but aparently forgot). Choosing the random geo/context names upfront makes the BuilderAndInfo go away and the whole thin slightly easier I think, but I'm fine either way. If you are interested, I tried something like [this](https://gist.github.com/cbuescher/34cf78ec6f10bab16fbb).